src/routers/mailingList/constant.ts

```diff
@@ -0,0 +1,20 @@
+export const EVENT_NAMES = [
+  'Games Night',
+  'Movie Night',
+  'Online Party',
+  'Scavenger Hunt',
+  'Talent Show',
+  'Study Buddies',
+  'Lewwe Talk',
+  'Trivia Night',
+  'Workout Sessions',
+  'Dance Sessions',
+];
+
+// Object with event names as keys and all values set to false
+export const DEFAULT_EVENT_RESPONSES = EVENT_NAMES.reduce<
```

Comment:
What made you decide to initialize it this way instead of 
```
export const DEFAULT_EVENT_RESPONSES = {
  'Games Night': false,
  'Movie Night': false,
...
}
```
Is there a typing benefit to doing it this way?

Reply:
Nope not really just that when/if the `EVENT_NAMES` array is updated this is automatically updated.

Reply:
Since we don't need the `EVENT_NAMES` array I'll change him over to this simpler object 👍